frontend/src/app/modules/shared/components/chart-switch-menu/chart-switch-menu.component.html

```diff
@@ -1,8 +1,13 @@
 <h1>
   <span *ngFor="let chart of chartSwitchMenu">
-    <span *ngIf="router.url.startsWith(chart.baseUrl); else showIcon">
-      {{ chart.label | translate }} (Dev)
+    <span *ngIf="router.url.startsWith(chart.baseUrl); else showDev">
+      {{ chart.label | translate }}
     </span>
```

Comment:
Condition in ngIf is always true.

Reply:
Yes, when navigating through the application this is true and desired.
But the charts in development like the time series chart under the url `/eventResultDashboardDev/showAll` will not be true and therefore it shows `(Dev)` in the title.

Reply:
You're right.